frontend/src/app/app.module.ts

```diff
@@ -25,7 +25,6 @@ import { AppRoutingModule } from './app-routing.module';
         ToastrModule.forRoot(),
         NgxChartsModule,
         SharedModule,
-        CoreModule,
```

Comment:
the same

Reply:
There is CoreModule referenced, it is just that eariler it was mentioned twice and now once